src/base_controller.coffee

```diff
@@ -14,8 +14,9 @@ class BaseController
   getActions: ({only, except} = {}) ->
     only = [].concat only ? []
     except = [].concat except ? []
-    for action of this
-      continue if BaseController::[action]?
+    for action in Object.getOwnPropertyNames @constructor.prototype
```

Comment:
Just curious what made you choose `Object.getOwnPropertyNames` over `Object.keys`